CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 A note on future revisions.  
   Everything within a major version number should be code compatible (with the exception of experimental interfaces).  The most notable example of an experimental interface is the support for multiple source inputs.  The APIs to deal with this will change in future minor releases.  Everything within a single minor release should be network compatible with other federates on the same minor release number.  Compatibility across minor release numbers may be possible in some situations but we are not going to guarantee this as those components are subject to performance improvements and may need to be modified at some point.  Patch releases will be limited to bug fixes and other improvements not impacting the public API or network compatibility.  Check the [Public API](./docs/Public_API.md) for details on what is included and excluded from the public API and version stability.
 
+## [2.4.1][] - 2020-03-04
+Increased code coverage and additional bug fixes.
+
+
+### Changed
+-   the helics webserver will build by default if the conditions are met
+
+### Fixed
+
+### Added
+-   `HELICS_DISABLE_WEBSERVER` option to turn off building of the webserver.  If is off by default so the webserver will build by default if Boost is enabled and is version 1.70 or higher
```

Comment:
```suggestion
-   `HELICS_DISABLE_WEBSERVER` option to turn off building of the webserver.  It will build by default if Boost is enabled and is version 1.70 or higher; otherwise it is off by default.
```